webapp/channels/src/selectors/rhs.ts

```diff
@@ -138,6 +138,37 @@ export function getIsSearchGettingMore(state: GlobalState): boolean {
     return state.entities.search.isSearchGettingMore;
 }
 
+export function makeGetDraft() {
```

Comment:
Why create a new selector for this instead of using the existing `getPostDraft` and `makeGetChannelDraft`?

Reply:
This is `makeGetChannelDraft` improved. I didn't want to remove the other one in case anything was depending on it, but now that I see, nobody else is importing it.

There are a few reasons to apply this improvement:
- Use the same function for channels and threads
- Ensure the root id and channel id are always set in the draft

The reason to do this function as a make instead of just a get, is because I want each textbox to handle their own default, so they don't step on each other channel ids or root ids.

Reply:
I didn't realize that they return a default object, so that makes sense to use a factory here.

To avoid making making too many more changes in this PR, we could do a followup to potentially get rid of `getPostDraft` and `makeGetChannelDraft` afterwards though